docs/source/weightvec.rst

```diff
@@ -74,10 +74,57 @@ vs
 
 :math:`s^2 = \frac{1}{n} \sum_{i=1}^n {\left({x_i - m}\right)^2 }`
 
-The unbiased estimate of the population variance, standard deviation or covariance is computed by replacing :math:`\frac{1}{\sum{w}}` with a factor dependent on the type of weights used:
+However, unbiased estimates (when ``corrected=true``) are dependent on the types of weights used. All weights presented here are a subtype of ``AbstractWeights``.
 
-- ``AnalyticWeights``: :math:`\frac{1}{\sum w - \sum {w^2} / \sum w}`
-- ``FrequencyWeights``: :math:`\frac{1}{\sum{w} - 1}`
-- ``ProbabilityWeights``: :math:`\frac{n}{(n - 1) \sum w}` where ``n`` equals `count(!iszero, w)`
+Weights
```

Comment:
It's weird to introduce the different kinds of weights inside the section on variance. I'd rather define them first in a separate section, and then explain what difference this makes for the correction of variance. I'm not even sure you need to copy all formulas: referring to the docstrings for `var` can be enough.